code/game/gamemodes/shadowling/shadowling_items.dm

```diff
@@ -100,3 +100,18 @@
 		td << "<span class='userdanger'>You enter the rift. Deafening chimes jingle in your ears. You are swallowed in darkness.</span>"
 	playsound(loc, 'sound/effects/EMPulse.ogg', 25, 1)
 	qdel(td)
+
+
+/obj/item/organ/internal/shadowtumor
+	name = "black tumor"
+	icon_state = "blacktumor"
+	origin_tech = "biotech=4"
```

Comment:
All you really need is 5 (which will get you to bio 6 for dem sexy Xray implants)